deps/wazuh_testing/wazuh_testing/tools/file.py

```diff
@@ -111,3 +117,55 @@ def write_json_file(file_path, data, ensure_ascii=False):
         escaped. If ensure_ascii is false, these characters will be output as-is.
     """
     write_file(file_path, json.dumps(data, indent=4, ensure_ascii=ensure_ascii))
+
+
+def check_file_exist(file_path):
+    return os.path.exists(file_path)
```

Comment:
I think it is not necessary to make this function since checking if a file exists or not in Python is just a single line. To avoid writing `os.path.exists` you can always import that function from the module. 
```python
from os.path import exists
# ...
exists(file_path)
```

Reply:
Done in 6bf7e6871c48474affeaa53d330e1682285d057f